docs/ref/contrib/postgres/aggregates.txt

```diff
@@ -58,14 +58,18 @@ General-purpose aggregation functions
     Returns ``True`` if at least one input value is true, ``None`` if all
     values are null or if there are no values, otherwise ``False``.
 
-``JsonAgg``
------------
+``JSONBAgg``
+------------
 
-.. class:: JsonAgg(expressions, **extra)
+.. class:: JSONBAgg(expressions, **extra)
 
     .. versionadded:: 1.11
 
-    Returns the input values as a ``JSON`` array.
+    Returns the input values as a ``JSON`` array.  PostgreSQL ships with two
```

Comment:
Do you think a single sentence would work here: "Returns ... using `JSONB_AGG()` (requires PostgreSQL 9.5+)."
I'm not sure if the other sentences are adding much.